### PR TITLE
Add multiplier for seeds and use it for articles and users

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

As a follow up to https://github.com/thepracticaldev/dev.to/pull/5948 I've added a "multiplication factor" variable for the seeds file.

You can use it like this:

```shell
$ SEEDS_MULTIPLIER=2 rails db:reset
$ psql PracticalDeveloper_development
PracticalDeveloper_development=# select count(*) from articles; select count(*) from users;
┌───────┐
│ count │
├───────┤
│    50 │
└───────┘
(1 row)

Time: 0.472 ms
┌───────┐
│ count │
├───────┤
│    20 │
└───────┘
(1 row)
```

:)

I haven't added it to the other entities yet, because I wanted to go about it in steps if this approach is deemed the correct one.

## Related Tickets & Documents

- https://github.com/thepracticaldev/dev.to/issues/4149

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# we use this to be able to increase the size of the seeded DB at will
+# eg.: `SEEDS_MULTIPLIER=2 rails db:seed` would double the amount of data
```
Comment (joshpuetz) [inline db/seeds.rb]: May want to spell out the base numbers here (10 users, 25 articles)

**Comment by nickytonline** [comment]
Also, the documentation would need to be updated as well, https://docs.dev.to/getting-started/db/#seed-data. It can come in another PR though.

**Comment by rhymes** [comment]
@nickytonline @joshpuetz @benhalpern @mstruve, thanks for the reviews! should we extend this to all the other entities in the file?

File: db/seeds.rb (hunk 1 above)
Comment (rhymes) [reply db/seeds.rb]: makes sense! I'll add it to the next PR :)

**Pull request merged**